gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ProjectInfo.java

```diff
@@ -324,7 +323,7 @@ public Builder toBuilder() {
       projectPb.setLifecycleState(state.toString());
     }
     if (createTimeMillis != null) {
-      projectPb.setCreateTime(ISODateTimeFormat.dateTime().print(createTimeMillis));
+      projectPb.setCreateTime(ISODateTimeFormat.dateTime().withZoneUTC().print(createTimeMillis));
```

Comment:
It was using our local time zone, so converting to and from milliseconds wasn't an issue.  I'll add a test that makes sure that the serialized time zone ends with a 'Z' for UTC